README.md

```diff
@@ -112,6 +112,8 @@ config: # Dict with all n8n config options
 #      postgresdb:
 #        database: n8n
 #        host: localhost
+#
+# existingSecret: "" # Use an existing Kubernetes secret, e.g created by hand or Vault operator.
```

Comment:
Am I right in the assumption that secret or existignSecret are **exclusive**, so you can't use both, right?
If that is the case, can you extend your comment highlighting, that you can either have `existingSecret` or `secret` but not both.